ui/raidboss/data/04-sb/trial/shinryu-ex.js

```diff
@@ -83,12 +84,14 @@
             en: 'Akh Morn on YOU',
             de: 'Akh Morn auf DIR',
             cn: '死亡轮回点名',
+            ko: '아크몬 대상자',
           };
         } else if (data.role == 'tank') {
           return {
             en: 'Akh Morn on ' + matches.target,
             de: 'Akh Morn auf ' + matches.target,
             cn: '死亡轮回点名',
```

Comment:
Could you please change this cn translation as `cn: '死亡轮回点' + data.ShortName(matches.target),` ?
I don't know why doesn't it include the target name, but it is just a minor comment so I don't want to create a new PR.

And another thing. As the localized things in other files, the target name we obtain from `matches` always be `data.ShortName(matches.target)`. Is this trial special or just forget to use the `data.ShortName` function?

Reply:
I don't know, I thought it was about character name having two word(?)
'Abc Qwe' to 'Qwe' thing?
Since Korean server has one word character name, I didn't care about it.

Reply:
Yes, in the international version, we all have first name and last name.
And I know players in Chinese server have only one-word name, too.
Thanks for your work, and let's see what would @quisquous say for this?

Reply:
Yes, I just forgot to add the ShortName here, sorry! Please add it in to everything.

Reply:
> add it in to everything
I thought this means every `matches.target` in every file I've made change with this PR and made another commit.